test/TritonGPU/loop-pipeline.mlir

```diff
@@ -93,910 +93,4 @@ tt.func @matmul_loop(%lb : index, %ub : index, %step : index,
   }
   tt.return %loop#2: tensor<128x128xf32, #C>
 }
-
-// CHECK-LABEL: tt.func @matmul_loop_nested
```

Comment:
Do you plan to erase the tests completely? Or will refactor them later?

Reply:
oops pushed that by accident, definitely not planning to erase those

Reply:
undid that